scripts/zkai_larger_native_boundary_candidate_selector_gate.py

```
@@ -0,0 +1,906 @@
+        raise LargerNativeBoundaryCandidateSelectorError("payload commitment drift")
+
+
+def mutate_source_artifact_digest(payload: dict[str, Any], artifact_id: str) -> None:
+    for artifact in payload["source_artifacts"]:
+        if artifact.get("id") == artifact_id:
+            artifact["sha256"] = "0" * 64
+            return
+    raise LargerNativeBoundaryCandidateSelectorError(f"missing source artifact {artifact_id}")
+
+
+def mutation_cases(payload: dict[str, Any]) -> list[tuple[str, Callable[[dict[str, Any]], None]]]:
```

Ruff (0.15.12)

[warning] Unused function argument: `payload`

(ARG001)